releases/8.1/languages/en.php

```diff
@@ -0,0 +1,89 @@
+
+    'final_class_constants_title' => 'Final class constants',
+    'final_class_constants_content' => '<p>It is possible to declare final class constants, so that they cannot be overridden in child classes.</p>',
+
+    'octal_numeral_notation_title' => 'Explicit Octal numeral notation',
+    'octal_numeral_notation_content' => '<p>It is now possible to write octal numbers with the explicit <code>0o</code> prefix.</p>',
+
+    'fibers_title' => 'Fibers',
+    'fibers_content' => '<p>Fibers are primitives for implementing lightweight cooperative concurrency. They are a means of creating code blocks that can be paused and resumed like Generators, but from anywhere in the stack. Fibers themselves don\'t magically provide concurrency, there still needs to be an event loop. However, they allow blocking and non-blocking implementations to share the same API.</p><p>Fibers allow getting rid of the boilerplate code previously seen with <code>Promise::then()</code> or Generator based coroutines. Libraries will generally build further abstractions around Fibers, so there\'s no need to interact with them directly.</p>',
+
+    'array_unpacking_title' => 'Array unpacking support for string-keyed arrays',
+    'array_unpacking_content' => '<p>PHP supported unpacking inside arrays through the spread operator before, but only if the arrays had integer keys. Now it is possible to unpack arrays with string keys too.</p>',
+
+    'performance_title' => 'Performance Improvements',
+    'performance_chart' => '<strong>Symfony Demo App request time</strong><br>
+                25 consecutive runs, 250 requests (sec)<br>
+                (less is better)<br>',
+    'performance_results' => 'The result (relative to PHP 8.0):<br />
+              - 3.5 % WordPress speedup<br />
+              - 27.0% Symfony Demo speedup',
+    'performance_related_functions_title' => 'Performance related features in PHP 8.1:',
+    'performance_jit_arm64' => 'JIT backend for ARM64 (AArch64)',
+    'performance_inheritance_cache' => 'Inheritance cache (avoid relinking classes in each request)',
+    'performance_fast_class_name_resolution' => 'Fast class name resolution (avoid lowercasing and hash lookup)',
+    'performance_timelib_date_improvements' => 'timelib and ext/date performance improvements',
+    'performance_spl' => 'SPL file-system iterators improvements',
+    'performance_serialize_unserialize' => 'serialize/unserialize optimizations',
+    'performance_internal_functions' => 'Some internal functions optimization (get_declared_classes(), explode(), strtr(), strnatcmp(), dechex())',
+    'performance_jit' => 'JIT improvements and fixes',
+
+    'bc_title' => 'Deprecations and backward compatibility breaks',
+    'bc_null_to_not_nullable' => 'Passing null to non-nullable internal function parameters is deprecated.',
+    'bc_return_types' => 'Return types in PHP built-in class methods and deprecation notices.',
```

Comment:
```suggestion
    'bc_return_types' => 'Tentative return types in PHP built-in class methods',
```
The "and deprecation notices" here didn't really make sense to me?